apps/nextjs/src/components/icons/picker/icon-picker.tsx

```diff
@@ -0,0 +1,118 @@
+import { useState } from "react";
+import {
+  Combobox,
+  Group,
+  Image,
+  InputBase,
+  Skeleton,
+  Text,
+  useCombobox,
+} from "@mantine/core";
+
+import { clientApi } from "@homarr/api/client";
+import { useScopedI18n } from "@homarr/translation/client";
+
+interface IconPickerProps {
+  initialValue?: string;
+  onChange: (iconUrl: string) => void;
+}
+
+export const IconPicker = ({ initialValue, onChange }: IconPickerProps) => {
+  const [value, setValue] = useState<string>(initialValue ?? "");
+  const [search, setSearch] = useState(initialValue ?? "");
+
+  const t = useScopedI18n("common");
+
+  const { data, isFetching } = clientApi.icon.findIcons.useQuery({
+    searchText: search,
+  });
+
+  const combobox = useCombobox({
+    onDropdownClose: () => combobox.resetSelectedOption(),
+  });
+
+  const notNullableData = data?.icons ?? [];
+
+  const totalOptions = notNullableData.reduce(
+    (acc, group) => acc + group.icons.length,
+    0,
+  );
+
+  const groups = notNullableData.map((group) => {
+    const options = group.icons.map((item) => (
+      <Combobox.Option value={item.url} key={item.id}>
+        <Group>
+          <Image src={item.url} w={20} h={20} />
+          <Text>{item.name}</Text>
+        </Group>
+      </Combobox.Option>
+    ));
+
+    return (
+      <Combobox.Group label={group.slug} key={group.id}>
+        {options}
+      </Combobox.Group>
+    );
+  });
+
+  return (
+    <Combobox
+      onOptionSubmit={(value) => {
+        setValue(value);
+        setSearch(value);
+        onChange(value);
+        combobox.closeDropdown();
+      }}
+      store={combobox}
+      withinPortal
+    >
+      <Combobox.Target>
+        <InputBase
+          rightSection={<Combobox.Chevron />}
+          value={search}
+          onChange={(event) => {
+            combobox.openDropdown();
+            combobox.updateSelectedOptionIndex();
+            setSearch(event.currentTarget.value);
+          }}
+          onClick={() => combobox.openDropdown()}
+          onFocus={() => combobox.openDropdown()}
+          onBlur={() => {
+            combobox.closeDropdown();
+            setSearch(value || "");
+          }}
+          rightSectionPointerEvents="none"
+          withAsterisk
+          label="Icon URL"
```

Comment:
Missing translation

Reply:
Overall translations are missing for apps / integrations pages
Created new issue #438